.github/workflows/pull_request_native.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       matrix:
         module: ["spring-integration", "quarkus-integration"]
-        java-version: [ 17, 21, 22 ] #Latest two LTS + latest non-LTS.
+        java-version: [ 21 ] # Latest LTS
```

Comment:
I wonder if it makes sense to do Windows and Mac here too.
I could argue that, for _native_, it makes more sense than for the JVM.